project.clj

```diff
@@ -1,48 +1,45 @@
-(defproject beop-grape "0.1.16"
-  :description "The opinionated, data-first, REST, GraphQL and Falcor enabled API Clojure library"
-  :url "https://github.com/cyppan/grape"
+(defproject beop-grape "0.2.0-SNAPSHOT"
+  :description "The opinionated, data-first, REST API Clojure library"
+  :url "https://github.com/BeOpinion/grape"
   :license {:name "Eclipse Public License"
             :url  "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.8.0"]
-                 [cheshire "5.7.0"]
-                 [clj-time "0.13.0"]
-                 [prismatic/schema "1.1.3"]
-                 [prismatic/plumbing "0.5.3"]
-                 [metosin/schema-tools "0.9.0"]
+  :dependencies [[org.clojure/clojure "1.10.3" :scope "provided"]
+                 [cheshire "5.10.0"]
+                 [clj-time "0.15.2"]
+                 [prismatic/schema "1.1.12"]
+                 [prismatic/plumbing "0.5.5"]
+                 [metosin/schema-tools "0.12.3"]
                  [com.novemberain/monger "3.5.0"]
-                 [org.clojure/core.match "0.3.0-alpha4"]
+                 [org.clojure/core.match "1.0.0"]
                  [ring-middleware-accept "2.0.3"]
-                 [bidi "2.0.16"]
-                 [org.clojure/core.async "0.2.395"]
+                 [bidi "2.1.6"]
+                 [org.clojure/core.async "1.3.610"]
                  [slingshot "0.12.2"]
-                 [org.clojure/tools.logging "0.3.1"]
-                 [org.slf4j/slf4j-api "1.7.23"]
-                 [org.slf4j/jul-to-slf4j "1.7.23"]
-                 [org.slf4j/log4j-over-slf4j "1.7.23"]
-                 [org.slf4j/jcl-over-slf4j "1.7.23"]
-                 [com.stuartsierra/component "0.3.2"]
-                 [http-kit "2.2.0"]
-                 [com.rpl/specter "1.1.0"]
+                 [com.taoensso/timbre "5.1.2"]
```

Comment:
Why used `timbre` instead of `clojure.tools.logging`?

Reply:
AFAIK we use timbre usually, I just switched to the logging api we already know.